resources/help.txt

```diff
@@ -112,10 +112,21 @@ Usage:
 
 Platform-specific usage:
     $ tns platform add android [--frameworkPath <File Path>] [--symlink]
-    $ tns platform add ios [--frameworkPath <File Path>] [--symlink] 
+    $ tns platform add android[@<Version>] [--frameworkPath <File Path>] [--symlink]
+    $ tns platform add ios [--frameworkPath <File Path>] [--symlink]
+    $ tns platform add ios[@<Version>] [--frameworkPath <File Path>] [--symlink]
 
 Configures the current project to target the selected platform. 
 
+<Version> is any available version published in npm.
```

Comment:
\<Version\> is any available version of the respective platform runtime published in npm. If @\<Version\> is not specified, the NativeScript CLI installs the latest stable runtime for the selected platform.
To list all available versions for android, run $ npm view tns-android versions
To list only experimental versions for android, run $ npm view tns-android dist-tags
To list all available versions for ios, run $ npm view tns-ios versions
To list only experimental versions for ios, run $ npm view tns-ios dist-tags